DeviceSecurityPkg/Library/SpdmSecurityLib/SpdmSecurityLibInterna.h

```diff
@@ -70,6 +70,8 @@ typedef struct {
 #define SPDM_DEVICE_CONTEXT_INSTANCE_SIGNATURE  SIGNATURE_32 ('S', 'D', 'C', 'S')
 #define SPDM_DEVICE_CONTEXT_INSTANCE_FROM_LINK(a)  CR (a, SPDM_DEVICE_CONTEXT_INSTANCE, Link, SPDM_DEVICE_CONTEXT_INSTANCE_SIGNATURE)
 
+#define PCR_INDEX_FOR_SIGNATURE_DB          1

```

Comment:
please add comment: current spec is using 7, but it will cause bitlocker fail. As such, we use 1.